README.md

```diff
@@ -73,6 +74,159 @@ Here is my post content.
 Note that we didn't add anything to the `scripts` block in the post. That's OK,
 content blocks without any content will be ignored.
 
+We can write multiple blocks in our page with the same block name:
+
+```html
+---
+layout: default
+---
+
+Here are the various academic qualifications I have been awarded over the years:
+
+{% contentfor qualification %}
+# Undergraduate Degree
+* University of Edinburgh
+* **Computational Phyics BSc**
+* **Graduated**: 2018
+* Grade: Upper Second with Honours
+{% endcontentfor %}
+
+
+{% contentfor qualification %}
+# Masters' Degree
+* University of Copenhagen
+* **Advanced Machine Learning**
+* **Graduated**: 2019
+* Grade: First with Honours
+{% endcontentfor %}
+
+
+{% contentfor qualification %}
+# Doctorate
+* University of Southampton
+* **Large-Scale Modelling using Neural Networks**
+* **Completed**: 2022
+{% endcontentfor %}
+```
+
+Then we can iterate over them in the layout file like so:
+
+```html
+<html>
+  <head>
+    {% contentblock scripts %}
+  </head>
+  <body>
+    <div class="heading">
+      {{ content }}
+    </div>
+    <hr/>
+    <div class="qualifications">
+      {% for qualification in page.contentblocks.qualifications %}
```

Comment:
I see above that the example uses `{% contentfor qualification %}` -- so in that case should this be `page.contentblocks.qualification`?